torch/_inductor/decomposition.py

```diff
@@ -447,11 +447,15 @@ def view_copy_dtype(self, dtype):
 
 @register_extra_random_decomp([aten.bernoulli_])
 def bernoulli_(self, p=0.5):
+    if self.device == torch.device("cpu"):
+        return NotImplemented
```

Comment:
```suggestion
        raise NotImplemented
```

Reply:
Its behavior is not as expected if we replace return with raise. I think we can align it with others in this file.
https://github.com/pytorch/pytorch/blob/ee6b19bd4c4f33590697f45dbd492e77dd9b3e34/torch/_inductor/decomposition.py#L249

Reply:
Oh, makes sense.